examples/trading/account_info.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+"""
+Example script showing how to get conditional orders for a wallet address using the Reya Trading SDK.
+
+Before running this example, ensure you have a .env file with the following variables:
+- PRIVATE_KEY: Your Ethereum private key
+- ACCOUNT_ID: Your Reya account ID
+- CHAIN_ID: The chain ID (1729 for mainnet, 89346162 for testnet)
+"""
+from dotenv import load_dotenv
+
+from reya_trading import ReyaTradingClient
+
+
+def main():
+    """Run the example to get conditional orders."""
+    # Load environment variables
+    load_dotenv()
+    
+    # Create a client instance with configuration from environment variables
+    client = ReyaTradingClient()
+    
+    # Check if we have a wallet address (either directly or derived from private key)
+    if not client.wallet_address:
+        print("Error: No wallet address found in environment variables.")
+        print("Please set either WALLET_ADDRESS or PRIVATE_KEY in your .env file.")
+        return
+    
+    # Show the wallet address we're using
+    print(f"Using wallet address: {client.wallet_address}")
+    
+    # Get conditional orders for the wallet
+    print("\n--- Getting conditional orders ---")
+    
+    try:
+        # Get all conditional orders for the wallet
+        conditional_orders = client.get_conditional_orders()
```

Comment:
i remember @danigr99727 was saying that we should avoid using conditional orders in our notation / terminology, shall we start cleaning up the notation here as well?